chapters/syntax.tex

```diff
@@ -398,7 +398,7 @@ \subsection{Expressions}\label{expressions1}
    | true
    | ( component-reference | der | initial | pure ) function-call-args
    | component-reference
-   | "(" output-expression-list ")" [ array-subscripts ]
+   | "(" output-expression-list ")" [ (array-subscripts | "." IDENT) ]
```

Comment:
In the grammar, we also have a space on the inside of parentheses (please also fix for `factor`):
```suggestion
   | "(" output-expression-list ")" [ ( array-subscripts | "." IDENT ) ]
```